src/sentry/api/helpers/deprecation.py

```diff
@@ -66,17 +66,19 @@ def _should_be_blocked(deprecation_date: datetime, now: datetime, key: str) -> b
             logger.exception("Invalid ISO8601 format for blackout duration")
             return False
 
-        if not croniter.is_valid(brownout_cron):
+        try:
+            # Move back one minute to check if the next iteration matches te
+            # current time
+            iter = CronSim(brownout_cron, now - timedelta(minutes=1))
+        except CronSimError:
             logger.error("Invalid crontab for blackout schedule")
             return False
 
-        # return True if now exactly matches the crontab
-        if croniter.match(brownout_cron, now):
+        if next(iter) == now.replace(second=0, microsecond=0):
```

Comment:
Cronsim doesn't have a `match` function, so we just move back one minute and iterate forward to see if we're at a matching time